proto/alliance.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package chremoas.esi;
+
+import "common.proto";
+
+service AllianceQuery {
+    rpc GetAlliances (EmptyRequest) returns (AlliancesResponse) {
+    };
+    rpc GetAllianceId (AllianceRequest) returns (AllianceIdResponse) {
+    };
+    rpc GetAllianceIdCorporations (AllianceRequest) returns (CorporationsResponse) {
+    };
+    rpc GetAllianceIdIcons (AllianceRequest) returns (AllianceIconResponse) {
+    };
+    rpc GetAllianceNames (AllianceNamesRequest) returns (AllianceNamesResponse) {
+    };
+}
+
+message AllianceRequest {
+    int32 AllianceId = 1;
+}
+
+message AllianceNamesRequest {
+    repeated int64 AllianceIds = 1;
```

Comment:
should be int32's?

Reply:
https://github.com/antihax/goesi/blob/master/esi/docs/AllianceApi.md#getalliancesnames says int64. that doesn't *feel* right, but I'm going to go with what the auto-generated docs say.

Reply:
makes sense, just wanted to make sure this wasn't an error.